requirements.txt

```diff
@@ -6,7 +6,7 @@ cryptography==3.3.2; python_version <= '2.7'
 cryptography==41.0.1; python_version > '3'
 fpdf>=1.7.2
 dbf>=0.88.019
-Pillow>=2.0.0
+Pillow<=9.5.0;
```

Comment:
Please open a separate Pull Request for this change, so we can merge it ASAP and fix the 32 bit build/release.
And limit pillow version just in case of 32 bits, for example with a condition `;platform_machine!='x86_64'`

Reply:
Yeah that's exactly what I did in one of the commits